web_src/js/index.js

```diff
@@ -1257,6 +1257,15 @@ function initPullRequestMergeInstruction() {
   });
 }
 
+function initRelease() {
+  $(document).on('click', '.remove-rel-attach', function() {
+    const uuid = $(this).data('uuid');
+    const id = $(this).data('id');
+    $(`input[name='attachment-del-${uuid}']`).attr('value', true);
```

Comment:
This is kinda odd, it'd be nicer to assemble a list on client side and put it into a single input

Reply:
And I also added `attachment-edit-${uuid}` to allow update attachment name.